README.md

```diff
@@ -80,6 +80,8 @@ $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
 
 ### Installing Mbed CLI
 
+Windows users can use the [mbed CLI for Windows installer](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/dev_tools/cli_install/) which will install mbed CLI and all necessary requirements with one installer.
```

Comment:
Could we change `which will` => `to`? I think it conveys the message better and is shorter.

Reply:
Please also change "mbed CLI" to "Mbed CLI" for proper branding.

Reply:
@AnotherButler @theotherjimmy Done with commit edac8c9 :)